solana/pyth2wormhole/client/src/config_file.rs

```diff
@@ -0,0 +1,34 @@
+#[derive(Deserialize, Serialize)]
+pub struct Config {
+    symbols: Vec<P2WSymbol>,
+}
+
+/// Config entry for a Pyth2Wormhole product + price pair
+#[derive(Deserialize, Serialize)]
+pub struct P2WSymbol {
+    /// User-defined human-readable name
```

Comment:
maybe leave a comment that this is purely to make the config file easier to understand

Reply:
Left a comment